tests/linearizability/failpoints.go

```diff
@@ -56,6 +56,8 @@ var (
 	CompactBeforeCommitBatchPanic            Failpoint = goPanicFailpoint{"compactBeforeCommitBatch", triggerCompact, AnyMember}
 	CompactAfterCommitBatchPanic             Failpoint = goPanicFailpoint{"compactAfterCommitBatch", triggerCompact, AnyMember}
 	RaftBeforeLeaderSendPanic                Failpoint = goPanicFailpoint{"raftBeforeLeaderSend", nil, Leader}
+	BlackholePeerNetwork                     Failpoint = blackholePeerNetworkFailpoint{duration: time.Second}
```

Comment:
nit: have you considered adding `target` (Leader, Follower, Quorum, All)? Similar to current functional test [cases](https://github.com/etcd-io/etcd/blob/main/tests/functional/rpcpb/rpc.proto#L431)

Reply:
For now I focused on getting proxy integrated into e2e. There are definitely couple of important cases in network functional failpoints that we should implement. Will add it to https://github.com/etcd-io/etcd/issues/14045